test_apps/advection/README.md

```diff
@@ -0,0 +1,56 @@
+# VAPOR Particle Advection Benchmarking
```

Comment:
Can you provide the command invocation that is needed to run the benchmark program?


Also, can you include the results from the benchmark (either in this readme or a separate file). The results should have all of the pertinent information such as:  the parameters used to configure the benchmark, the host name where the benchmark was run, and any other information that might be needed to recreate the results.

Reply:
Ok, I'll update all that information too.